forked-daapd.conf.in

```diff
@@ -157,6 +157,9 @@ library {
 	# to trigger a rescan.
 #	filescan_disable = false
 
+	# Extra tags should be searched in radio streams for artwork
```

Comment:
Add a reference to the excellent doc you made in the README

Reply:
Done